### PR TITLE
fix: help center protection

### PR DESCRIPTION
**Pull request by ZEED2468**
### PR Description:

- [x] Added SuperAdminGuard to protect edit and delete help center routes.

### CODE REVIEW

**Review by olika-ashley — APPROVED** [review]

**Comment by ZEED2468** [comment]
I don't understand what you mean should a user be able to delete an help center topics 
@incredible-phoenix246

**Comment by TheCodeGhinux** [comment]
@ZEED2468  don't worry about it.

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
       "hasInstallScript": true,
       "license": "UNLICENSED",
       "dependencies": {
-        "@css-inline/css-inline-linux-x64-gnu": "^0.14.1",
         "@css-inline/css-inline": "^0.14.1",
         "@faker-js/faker": "^8.4.1",
```
Comment (TheCodeGhinux) [inline package-lock.json:14]: Don't push your package-lock

File: .gitignore [hunk 2]
```diff
@@ -388,6 +388,7 @@ Temporary Items
 =======
 # Local
 dist
+package-lock.json
```
Comment (Cobidex) [inline .gitignore:391]: Please don't add this to gitignore. Remove

**Review by Cobidex — APPROVED** [review]

**Pull request merged**